### DISCUSSION TITLE
New installation boots into console only

### DISCUSSION THREAD

**Discussion by NickJH** [rern/rAudio#1075] [General]
Hi, Newbe here.
I am just trying to try rAudio on an old Pi B+ that I have lying around. I have installed the rpi0 image and booted the Pi (with a wired connection for the moment), but it just boots into the console rather than Gui and I can see the startup service has failed.

I have ssh'd in and I cannot start the service manually but systemctl does not tell me anything:
```
rAudio:~  #  systemctl status startup
x startup.service - rAudio Startup
     Loaded: loaded (/etc/systemd/system/startup.service; enabled; vendor preset: disabled)
     Active: failed (Result: exit-code) since Sat 2022-10-15 16:02:28 UTC; 15s ago
    Process: 808 ExecStart=/srv/http/bash/startup.sh (code=exited, status=1/FAILURE)
   Main PID: 808 (code=exited, status=1/FAILURE)
        CPU: 12.430s

Oct 15 16:02:07 rAudio systemd[1]: Starting rAudio Startup...
Oct 15 16:02:28 rAudio systemd[1]: startup.service: Main process exited, code=exited, status=1/FAILURE
Oct 15 16:02:28 rAudio systemd[1]: startup.service: Failed with result 'exit-code'.
Oct 15 16:02:28 rAudio systemd[1]: Failed to start rAudio Startup.
Oct 15 16:02:28 rAudio systemd[1]: startup.service: Consumed 12.430s CPU time.
```
And in journalctl:
```
Oct 15 16:08:22 rAudio systemd[1]: Starting rAudio Startup...
-- Subject: A start job for unit startup.service has begun execution
-- Defined-By: systemd
-- Support: https://lists.freedesktop.org/mailman/listinfo/systemd-devel
--
-- A start job for unit startup.service has begun execution.
--
-- The job identifier is 594.
Oct 15 16:08:43 rAudio systemd[1]: startup.service: Main process exited, code=exited, status=1/FAILURE
-- Subject: Unit process exited
-- Defined-By: systemd
-- Support: https://lists.freedesktop.org/mailman/listinfo/systemd-devel
--
-- An ExecStart= process belonging to unit startup.service has exited.
--
-- The process' exit code is 'exited' and its exit status is 1.
Oct 15 16:08:43 rAudio systemd[1]: startup.service: Failed with result 'exit-code'.
-- Subject: Unit failed
-- Defined-By: systemd
-- Support: https://lists.freedesktop.org/mailman/listinfo/systemd-devel
--
-- The unit startup.service has entered the 'failed' state with result 'exit-code'.
Oct 15 16:08:43 rAudio systemd[1]: Failed to start rAudio Startup.
-- Subject: A start job for unit startup.service has failed
-- Defined-By: systemd
-- Support: https://lists.freedesktop.org/mailman/listinfo/systemd-devel
--
-- A start job for unit startup.service has finished with a failure.
--
-- The job identifier is 594 and the job result is failed.
Oct 15 16:08:43 rAudio systemd[1]: startup.service: Consumed 12.397s CPU time.
-- Subject: Resources consumed by unit runtime
-- Defined-By: systemd
-- Support: https://lists.freedesktop.org/mailman/listinfo/systemd-devel
--
-- The unit startup.service completed and consumed the indicated resources.
```
I am not sure either are very useful. I have tried rebooting. How can I proceed?

**Comment by NickJH** [comment]
One thing I have noticed is there is a `/` missing in line 149 of /srv/http/bash/startup.sh, so not:
```
	$dirbash/cmd-list.sh &> dev/null &
```
but:
```
	$dirbash/cmd-list.sh &> /dev/null &
```

However, fixing this does not help the service start

**Reply by Sagnasty** [reply]
@NickJH 

You may have used an incorrect image - first identify which RPI model you are using (zero, 1, 2, 3, or 4).

You can find all images here: [https://github.com/rern/rAudio-1/releases](url)

![image](https://user-images.githubusercontent.com/76575081/196010167-d61de596-737b-42c5-bcb8-6641470f371e.png)

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Comment by NickJH** [comment]
Thanks for the reply. I am pretty sure I have the right image (https://github.com/rern/rAudio-1/releases/download/i20220904/rAudio-1-RPi0-1-20220904.img.xz) as I have a Pi B+:
```
rAudio:~  #  cat /proc/cpuinfo
processor       : 0
model name      : ARMv6-compatible processor rev 7 (v6l)
BogoMIPS        : 697.95
Features        : half thumb fastmult vfp edsp java tls
CPU implementer : 0x41
CPU architecture: 7
CPU variant     : 0x0
CPU part        : 0xb76
CPU revision    : 7

Hardware        : BCM2835
Revision        : 000e
Serial          : 00000000cdcbbfe0
Model           : Raspberry Pi Model B Rev 2
```
I have double-checked the image I downloaded and it appears to be correct. I can try again later today.

I do notice I can still access the server using the Android app or the web browser, so something is going right.

As a totally separate question (I can post again), how do I add a DLNA source? All my music files are on a Plex server.

**Comment by rern** [comment]
Chromium browser is too much for Raspberry Pi Zero and 1. It's omitted by default.
(A note has been added to the first page.)

**Comment by NickJH** [comment]
Oh OK. I didn't understand the implication of that. No problem as I intend to run headless.

It also looks like rAudio cannot work as a DLNA client so I am looking to use a Samba share instead.
